service/ocean/providers/aws/migrationsummary.go

```diff
@@ -0,0 +1,97 @@
+package aws
```

Comment:
please add get migration status as well, if it's not implemented already

Reply:
Added implementation for get migration status as well.